components/prism-tt2.js

```diff
@@ -0,0 +1,61 @@
+(function(Prism) {
+
+	Prism.languages.tt2 = Prism.languages.extend('clike', {
+		comment: {
+			pattern: /#.*|\[%#[\s\S]*?%\]/,
+			lookbehind: true
+		},
+		keyword: /\b(?:BLOCK|CALL|CASE|CATCH|CLEAR|DEBUG|DEFAULT|ELSE|ELSIF|END|FILTER|FINAL|FOREACH|GET|IF|IN|INCLUDE|INSERT|LAST|MACRO|META|NEXT|PERL|PROCESS|RAWPERL|RETURN|SET|STOP|TAGS|THROW|TRY|SWITCH|UNLESS|USE|WHILE|WRAPPER)\b/,
+		punctuation: /[[\]{},()]/
+	});
+
+	delete Prism.languages.tt2['operator'];
+	delete Prism.languages.tt2['variable'];
+	Prism.languages.insertBefore('tt2', 'number', {
+		operator: /(?:=>|==|!=|<=|<|>=|>|=|&&|\|\||\||!|and|or|not)/,
+		variable: {
+			pattern: /[a-z]\w*(?:\s*\.\s*(?:\d+|\$?[a-z]\w*))*/i,
+			greedy: true
```

Comment:
Do variables really need to be greedy? The `greedy` behaviour has a bit of a performance hit, so you should make sure you actually require it.

Reply:
With the `\b` around the keyword-like operators `and|or|not` it is no longer needed.  Thanks for the hint.